### PR TITLE
Add GSoC 2020 accepted projects announcement blog post

### PR DESCRIPTION
**Pull request by vogelpi**

### CODE REVIEW

File: site/content/blog/gsoc-2020-accepted-projects-announcement.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
++++
+
+We are pleased to announce that we will be mentoring two students as part of Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/).
+
+We are looking forward to working with Flavien and Yuichi on features and tools to improve IP such as Ibex, our open-source RISC-V core.
+
+## Flavien Solt: Simulated Memory Controller
+
+It is a common pitfall to misinterpret or incorrectly scale performance numbers derived from benchmarks run on an FPGA-based SoC design.
+The problem is that your external memory interface is running at a very high speed compared to the core CPU (e.g. a 25MHz core clock speed but external memory running a several hundred MHz).
```
Comment (asb) [inline site/content/blog/gsoc-2020-accepted-projects-announcement.md:14]: your => the

File: site/content/blog/gsoc-2020-accepted-projects-announcement.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+This can be misleading when trying to consider what the performance would be on an ASIC, as the CPU clock speed could be many times higher but the memory frequency remains the same or increases by a much smaller amount.
+The solution is to have a simulation-ready memory controller that will produce delays more representative of a system where the memory interface is running at a much slower speed.
```
Comment (asb) [inline site/content/blog/gsoc-2020-accepted-projects-announcement.md:16]: have a simulation-ready => implement a "simulated"

Comment (asb) [reply site/content/blog/gsoc-2020-accepted-projects-announcement.md:16]: Actually Rob suggests instead 'The solution is to implement a memory controller for the FPGA that can accurately "simulate" memory access delays in terms of the processor's clock cycles.'

File: site/content/blog/gsoc-2020-accepted-projects-announcement.md (hunk 1 above)
Comment (vogelpi) [reply site/content/blog/gsoc-2020-accepted-projects-announcement.md]: That's fixed.

File: site/content/blog/gsoc-2020-accepted-projects-announcement.md (hunk 2 above)
Comment (vogelpi) [reply site/content/blog/gsoc-2020-accepted-projects-announcement.md]: I went for Rob's solution.

**Comment by vogelpi** [comment]
Thanks for your approval @asb . I implemented the suggestions. Will merge after the CI checks passed.

**Pull request merged**